src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="http://google.pl"
+      data-qa="big"
+    >
+      <label
+        class="label"
+        for="search">
+        <img
+           src="images/Search.svg"
+           alt="search icon"
+           class="search-icon"
+           id="search"
```

Comment:
This id is not needed, you need to set id for input element so the label is connected to it.